homework/python_processing.py

```diff
@@ -0,0 +1,174 @@
+# Import libraries
+import pandas as pd
+
+
+# Import datasets
+employees=pd.read_csv("./extracts/employees.csv")
+products=pd.read_csv("./extracts/products.csv")
+orders=pd.read_csv("./extracts/orders.csv")
+
+
+# Inspect datasets
+print("Employees")
+print(employees.shape)
+print(employees.head())
+
+print("Products")
+print(products.shape)
+print(products.head())
+
+print("Orders")
+print(orders.shape)
+print(orders.head())
+
+
+# Check some hypotheses regarding how to put the tables in 3NF format
+print(products.groupby('product_line').product_code.nunique()) # This is linked to the product code
+
+print(products.groupby('html_description').product_code.nunique()) # Every value is NaN, we might drop this variable?
```

Comment:
Clean up any prints that you don't think are useful for running this script "in production" -- in other words, printing out diagnostics like above might be nice! But these look like they are for your exploration only.